src/main/java/sx/blah/discord/api/internal/DiscordUtils.java

```diff
@@ -549,6 +550,7 @@ public static IChannel getChannelFromJSON(IShard shard, IGuild guild, ChannelObj
 				channel.roleOverrides.clear();
 				channel.userOverrides.putAll(overrides.getLeft());
 				channel.roleOverrides.putAll(overrides.getRight());
+				channel.setCategoryId(categoryId);
```

Comment:
The method and variable should have a capital `D` for ID as anything else would be inconsistent with the rest of the source.